### PR TITLE
Add progress bars and move messages to separate file

### PR DESCRIPTION
**Pull request by mjarvis1997**
hope you like it!

image

### CODE REVIEW

**Comment by mjarvis1997** [comment]
by the way, worked fine for me on linux

**Comment by carter-reynolds** [comment]
yo this is sick!

File: game.py [hunk 1]
```diff
@@ -38,7 +39,9 @@
 
 ply = {
     "energy": 24,
-    "hp": 100
+    "max_energy": 24,
```
Comment (carter-reynolds) [inline game.py:42]: good thinking ahead on these. you can tell i was thinking about min max values with the eventual thoughts of having buffs from items or purchased abilities or something.

File: game.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 import time
 from termcolor import colored, cprint # pip3 install termcolor
 from playsound import playsound       # pip3 install playsound
+from messages import random_rest_messages, random_mine_messages, random_fish_messages, random_combat_messages
```
Comment (carter-reynolds) [inline game.py:5]: ah okay so that's how that works. I tried that with a lot of the global variables before i changed them to dictionaries. thanks

**Review by carter-reynolds — APPROVED** [review]

**Pull request merged**